tests/filesystem/is_absolute_test.php

```diff
@@ -1,68 +0,0 @@
-<?php
```

Comment:
I think you should keep both the old and new tests (even if they look duplicated)